my-app/src/Container/chat/servises/messagesService.js

```diff
@@ -0,0 +1,82 @@
+import { PERSONS, FIRST_MESSAGES, RESPONSE_MESSAGES } from "../../constants/constants.js";
+
+// export function sendMyself(e) {
+// 	return new Promise((resolve) => {
+// 		resolve(
+// 			(myselfMessages = [
+// 				...myselfMessages,
+// 				{ TEXT_MESSAGE: e, DATE_MESSAGE: Date.now(), SENDER: "myself" },
+// 			])
+// 		);
+// 	});
+// }
+
+// export function responseInterlocator() {
+// 	return new Promise((resolve) => {
+// 		resolve(
+// 			(iterlocutorMessages = [
+// 				...iterlocutorMessages,
+// 				{
+// 					TEXT_MESSAGE:
+// 						RESPONSE_MESSAGES[Math.floor(Math.random() * RESPONSE_MESSAGES.length)],
+// 					DATE_MESSAGE: Date.now(),
+// 					SENDER: "interlocutor",
+// 				},
+// 			])
+// 		);
+// 	});
+// }
+
+// export function resetValue() {
+// 	return new Promise((resolve) => {
+// 		resolve((myselfMessages = []), (iterlocutorMessages = []));
+// 	});
+// }
+
+class MessagesService {
+	myselfMessages = [];
```

Comment:
why you need two arrays for messages?

Reply:
because I tried to simulate sending messages and receiving messages. In order not to complicate the code, since the universities of the class functions occurs through "asynx" and "await", so as not to use the same variable. Sender and recipient messages are separated into different variables